noxfile.py

```diff
@@ -34,10 +34,11 @@ def get_versions_from_pypi(package_name: str) -> List[str]:
     response = requests.get(f"https://pypi.org/rss/project/{package_name}/releases.xml")
     response.raise_for_status()
     xml_tree = ElementTree.fromstring(response.text)
-    versions = [i.text for i in xml_tree.findall("channel/item/title") if i.text]
-    # Verify all strings have the format for version
-    [Version(version) for version in versions]
```

Comment:
Why did you delete it?

Reply:
Because that validation provides no value that I can speak of, and it will instead risk of breaking with non-Python packages. Remember, that `Version` object has some _strong_, PEP-related assumptions.